client/app/bundles/course/duplication/components/IndentedCheckbox.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC, ReactNode } from 'react';
+import { Checkbox, FormControlLabel } from '@mui/material';
+
+import { indentClassName } from '../constants';
+
+interface Props {
+  indentLevel: number;
+  children?: ReactNode;
+  label: ReactNode;
+  checked?: boolean;
+  disabled?: boolean;
+  onChange?: (
+    event: React.ChangeEvent<HTMLInputElement>,
+    value: boolean,
+  ) => void;
+}
+
+const IndentedCheckbox: FC<Props> = (props) => {
+  const { indentLevel, children, label, checked, disabled, onChange } = props;
+
+  return (
+    <div className="flex items-center">
+      <FormControlLabel
+        className="w-auto mx-0 space-y-0"
+        control={
+          <Checkbox
+            checked={checked}
+            className={`${indentLevel > 0 && indentClassName[indentLevel]} ${children && 'w-auto'}`}
```

Comment:
Why is there a need to check `indentLevel > 0` when `indentClassName[0]` is defined?

Reply:
sorry for the oversight. At first I didn't define `indentClassName` for 0 (since there should not be any margin for 0 indent level). However later on, I added the definition but forgot to change this syntax.